driver/result_set.h

```diff
@@ -17,14 +17,15 @@ extern const std::string::size_type initial_string_capacity_g;
 class ColumnInfo {
 public:
     void assignTypeInfo(const TypeAst & ast);
-    void updateTypeId();
+    void updateTypeInfo();
 
 public:
     std::string name;
     std::string type;
     std::string type_without_parameters;
     DataSourceTypeId type_without_parameters_id = DataSourceTypeId::Unknown;
-    std::size_t display_size = 0;
+    std::int64_t display_size = SQL_NO_TOTAL;
+    std::size_t display_size_so_far = 0;
```

Comment:
please add a comment explaining why do you need this field.

Reply:
Added comment. 0 doesn't have any special meaning.